### ISSUE TITLE
Incomplete explanation of coalesce function

### ISSUE THREAD

**Issue by matthew25187** [MicrosoftDocs/azure-devops-docs#13611]
Documentation for the `coalesce` function appears to be incomplete.  First, this line:

> Evaluates the parameters in order, and returns the value that does not equal null or empty-string.

Given it is stated that the parameters are evaluated "in order" (I'm assuming from left to right), it would be logical to expect that the _first_ non-null and not empty argument is returned.  Therefore, I'm assuming this should read (emphasis mine):

> Evaluates the parameters in order, and returns the **first** value that does not equal null or empty-string.

Second, why is there no explanation of what the function returns when _all_ of the listed values are null or empty (string)?

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 77c58a78-a567-e99a-9eb7-62dddd1b90b6
* Version Independent ID: 680a79bc-11de-39fc-43e3-e07dc762db18
* Content: [Expressions - Azure Pipelines](https://learn.microsoft.com/en-us/azure/devops/pipelines/process/expressions?view=azure-devops)
* Content Source: [docs/pipelines/process/expressions.md](https://github.com/MicrosoftDocs/azure-devops-docs/blob/main/docs/pipelines/process/expressions.md)
* Service: **azure-devops-pipelines**
* Sub-service: **azure-devops-pipelines-process**
* GitHub Login: @juliakm
* Microsoft Alias: **jukullam**

**Comment by WilliamAntonRohm** [comment]
@matthew25187 -- Matthew, thank you for your cogent feedback.  Please consider these resources:
- [Azure DevOps Services](https://azure.microsoft.com/support/devops/) 
- [Azure DevOps on Stack Overflow](https://stackoverflow.com/questions/tagged/azure-devops)

@juliakm -- Julia, please look into this issue.

**Comment by juliakm** [comment]
@matthew25187 You are right on both your comments. The expression returns the first non-empty value left-to-right. When every value is empty, nothing is returned. I'll update the article.
